<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrainModelMovementDialog</class>
 <widget class="QDialog" name="TrainModelMovementDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>210</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>210</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>200</width>
    <height>210</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Movement Info</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Commanded Power: 100W</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Actual Power: 100W</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Commanded Speed: 46 Mph</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Actual Speed: 38 Mph</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Authority: 3</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
